src/backend/joanie/core/utils/sentry.py

```diff
@@ -0,0 +1,15 @@
+"""Sentry utilities."""
+from django.conf import settings
+from cryptography.fernet import Fernet
+import json
+from django.core.serializers.json import DjangoJSONEncoder
+
+def encrypt_context(event, hint):
+    """Encrypt the context of the event sent to Sentry."""
+    context = event.get("extra", {}).get("context")
+    key = settings.LOGGING_SECRET_KEY
+    cipher_suite = Fernet(key)
+    context = json.dumps(context, cls=DjangoJSONEncoder).encode()
```

Comment:
We could use our custom JSON Encoder to support money field ?

Reply:
Nice idea :+1: